packages/cli/src/zosfiles/compare/ds/Dataset.handler.ts

```diff
@@ -9,114 +9,46 @@
 *
 */
 
-import { AbstractSession, IHandlerParameters, ITaskWithStatus, TaskStage, DiffUtils } from "@zowe/imperative";
+import { AbstractSession, IHandlerParameters, ITaskWithStatus, TaskStage } from "@zowe/imperative";
 import { Get, IZosFilesResponse } from "@zowe/zos-files-for-zowe-sdk";
 import { ZosFilesBaseHandler } from "../../ZosFilesBase.handler";
+import CompreBaseHelper from '../CompareBaseHelper';
```

Comment:
Please fix the typo and use a non-default import:
`import { CompareBaseHelper } from '../CompareBaseHelper';`